src/barrier-picker/BarrierCard.js

```diff
@@ -16,11 +17,10 @@ export default class BarrierCard extends PureComponent {
         barrierInfo: PropTypes.object,
         barrierType: PropTypes.oneOf(['relative', 'absolute']),
         isIntraDay: PropTypes.bool,
-        onBarrier1Change: PropTypes.func,
-        onBarrier2Change: PropTypes.func,
-        onError: PropTypes.func,
         pipSize: PropTypes.number,
         spot: PropTypes.number,
+        actions: PropTypes.object,
```

Comment:
Now we do not pass actions as props, instead import them, refer to how we get actions in `TradeParams`